arith/gf2_128/src/gf2_ext128/neon.rs

```diff
@@ -394,3 +403,41 @@ pub(crate) fn mul_by_x_internal(a: &uint32x4_t) -> uint32x4_t {
         vreinterpretq_u32_u64(res)
     }
 }
+
+impl SimdField for NeonGF2_128 {
```

Comment:
Same problem here. SimdField should not be implemented for this.

Reply:
reverted by https://github.com/PolyhedraZK/Expander/pull/142/commits/4fd26abc44b17fa828b34d4bccf3a2eba86029d4